components/integration/production/stone-prod-p01/kustomization.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+- ../base
+components:
+  - ../../rh-certs
```

Comment:
I can't find the files under `../../rh-certs`, am I wrong? Thanks!

Reply:
![image](https://github.com/redhat-appstudio/infra-deployments/assets/20207783/e9e86365-ed88-4c69-b069-a64483331150)
I see them there

Reply:
Ah, yes, thanks. I found it.